src/builder/conditional.rs

```diff
@@ -197,7 +196,7 @@ impl CaseBuilder<Hugr> {
             signature: signature.clone(),
         };
         // TODO: Allow input extensions to be specified
```

Comment:
Is the todo still valid if the extension set is open?

Reply:
Nope, I'll get rid of it!